simulator/tools/maxmin_field.py

```diff
@@ -29,31 +29,31 @@
 
 
 MAX_VALUES = {
-    'longitude': 180,
-    'latitude': 90,
-    'radius': 1000,
-    'length': 1000,
-    'width': 1000,
-    'height': 1000,
-    'expiration_duration': 100,
-    'idle_duration': 16777215,
-    'running_duration': 16777215,
-    'temperature': 50,
-    'current_sequence': 7,
-    'total_sequences': 7,
-    'priority': 255,
-    'pressure': 20460,
-    'time_to_flat': 2097151,
-    'light_intensity': 1275,
-    'hours': 24,
+    "longitude": 180,
```

Comment:
Why these changes? Is there something with the single quote that is causing issues?

Reply:
This was causing the lint issues. In Python, both single and double quotes work, but since we've configured Flake8 for linting, it suggests using double quotes.